pkg/osquery/extension.go

```diff
@@ -219,8 +229,100 @@ func (e *Extension) getHostIdentifier() (string, error) {
 	return IdentifierFromDB(e.db)
 }
 
+func SetupLauncherKeys(db *bbolt.DB) error {
+	err := db.Update(func(tx *bbolt.Tx) error {
+
+		b, err := tx.CreateBucketIfNotExists([]byte(configBucket))
```

Comment:
can we name this variable `bucket`? By the time I made it to the end of the function I forgot what `b` was =)

Reply:
Sure